### ISSUE TITLE
Chrome extension>UI clipping with DPI above 100%

### ISSUE THREAD

**Issue by metaltermite** [goldenratio/xbmc-web-remote#37]
When DPI scaling is above 100% (125% in my use-case), the bottom portion of the visible Chrome extension is clipped - specifically, the text labels for the bottom bottoms. See picture attachment.

![2016-01-02](https://cloud.githubusercontent.com/assets/10378599/12076738/daaa856c-b186-11e5-811f-63745d0d8e2a.png)

**Comment by goldenratio** [comment]
umm.. I am on Windows 10, not able to reproduce it,

![image](https://cloud.githubusercontent.com/assets/3019293/12078125/6efbda44-b20f-11e5-9476-ad207f6a8766.png)

![image](https://cloud.githubusercontent.com/assets/3019293/12078130/894905a2-b20f-11e5-853f-79d07edaee91.png)

**Comment by goldenratio** [comment]
any updates on this? still reproducible?

**Comment by metaltermite** [comment]
Still the same. Does PPI matter? Mine is 141.

**Comment by goldenratio** [comment]
don't know.. how can I change PPI?
Probably could be chrome issue.

**Comment by metaltermite** [comment]
I don't think you can change PPI, assuming that's a cause. I've tried uninstalling and reinstalling Kassi. Disabled other extensions that may be affecting it. No change. It does work properly on Firefox. 

Chrome Version 47.0.2526.106 m
Windows 10 x64
Kassi (for Chrome) Version: 2.3.3
